src/99-appendices/06-meg-file-formats.md

```diff
@@ -52,21 +52,44 @@ To learn more about CTF’s data organization:
 
 ## Neuromag/Elekta/MEGIN
 
-Neuromag/Elekta/MEGIN data and Tristan Technologies BabyMEG data is stored with
-file extension `.fif`. The digitized positions of the head points are saved
-inside the fif file along with the MEG data, with typically no `_headshape`
-file.
+Neuromag/Elekta/MEGIN and Tristan Technologies BabyMEG data is stored as
+FIFF files with the extension `.fif`. The digitized positions of the head
+points are saved inside the fif file along with the MEG data, with typically no
+`_headshape` file.
 
 ```Text
 sub-<label>[_ses-<label>]_task-<label>[_run-<index>]_meg.fif
 ```
 
-Note that we do not provide specifications for cross-talk and
-fine-calibration matrix files in the current version of BIDS.
+### Cross-talk and fine-calibration files
 
-Example:
+Raw FIFF files typically need to be processed using Maxwell filtering
```

Comment:
Please change to "In case internal active shielding (IAS) is used during acquisition, raw FIFF files need to be processed using  ..."

In case IAS is not used, Maxfilter is not required but only optional.

Reply:
Addressed in 8c17eaf